core/rawdb/accessors_state.go

```diff
@@ -256,11 +256,31 @@ func ReadStateHistory(db ethdb.AncientReaderOp, id uint64) ([]byte, []byte, []by
 // history starts from one(zero for initial state).
 func WriteStateHistory(db ethdb.AncientWriter, id uint64, meta []byte, accountIndex []byte, storageIndex []byte, accounts []byte, storages []byte) {
 	db.ModifyAncients(func(op ethdb.AncientWriteOp) error {
-		op.AppendRaw(stateHistoryMeta, id-1, meta)
-		op.AppendRaw(stateHistoryAccountIndex, id-1, accountIndex)
-		op.AppendRaw(stateHistoryStorageIndex, id-1, storageIndex)
-		op.AppendRaw(stateHistoryAccountData, id-1, accounts)
-		op.AppendRaw(stateHistoryStorageData, id-1, storages)
+		err := op.AppendRaw(stateHistoryMeta, id-1, meta)
+		if err != nil {
```

Comment:
Why did the community ignore this error? 
Why can't we ignore it?

Reply:
Maybe they haven't encountered this problem yet? 

And  we also ignore it. See db.ModifyAncients and WriteStateHistory. This error is not passed upwards